downstream/modules/platform/proc-controller-image-pull-policy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="proc_controller-image-pull-policy_{context}"]
+
+= controller-image-pull-policy
+
```

Comment:
I would suggest adding an abstract of some type to this procedure because the "why" for this topic is not clear. See the current section or sub-chapter "3.2.1 Configure your automation controller operator route options" (although this title displays another inconsistency in our docs with regards to modularization rules; it should be "Configuring...").  In another example, sub-chapter "3.2 Installing the automation controller operator" doesn't necessarily need an abstract because you understand why it's necessary, if that makes sense.

Reply:
I would dare to guess that anyone setting up an image pull policy understands it is the current image they are pulling, none, always, or IfNotPresent, which means the image gets pulled if it is not already on the node. These are Kubernetes rules, not Red Hat's, so most people familiar with Kubernetes, and this I gather from conversations I have had, would know these rules and set them accordingly.